### ISSUE TITLE
Auto download/star based on a podcast title/description in the user's feed containing a previously saved string

### ISSUE THREAD

**Issue by ciaranmag** [Automattic/pocket-casts-android#434]
### What problem does this address?

To save reading the description of many episodes I'd like episodes (within my feed) to automatically download or star if the podcast title or description contains a string/regex expression which I saved earlier (like a certain person's name or a company name or a specific topic)

### What is your proposed solution?

Save strings of text or regex expressions e.g. a person's name, a company. Associate an action with each string e.g. auto download/star.

When that text appears in a podcast episode's title or description then the episode will be automatically downloaded or starred (user's preselected choice)

### Did you search for existing feature suggestions?

- [X] I have searched for existing feature suggestions.

**Comment by CookieyedCodes** [comment]
Agreed & I personally would also like this as a feature of the local auto archive settings, So I can archive encore episodes based on the title/shownotes so I don't waste time on stuff I've allready listened too🤔 😉
